cuebot/src/test/java/com/imageworks/spcue/test/dao/postgres/HostDaoTests.java

```diff
@@ -82,6 +81,10 @@ public class HostDaoTests extends AbstractTransactionalJUnit4SpringContextTests
 
     public HostDaoTests() { }
 
+    // Hardcoded value of dispatcher.memory.mem_reserved_system
+    // to avoid having to read opencue.properties on a test setting
+    private final long MEM_RESERVED_SYSTEM = 524288;
```

Comment:
Recommendation:

Better move the hardcoded to the cuebot/src/test/resources/opencue.properties

A hardcoded constant makes sense to ensure that tests remain stable and do not depend on external configurations. However, if future testing requires different memory values, this approach may limit flexibility. Consider allowing this value to be overridden in test settings by an environment variable or through dependency injection in the future.

Reply:
Noted. But for the urgency of this change, I will add a TODO for the suggestion to be implemented in the future.

Reply:
It's also worth noting that None of the project unit tests read from opencue.properties, so I'm just following the practice employed by previous contributors